packages/bulkapi/src/database/QueryMaker.ts

```diff
@@ -376,6 +145,9 @@ export class QueryMaker {
     }
 
     public findNodeIdsInUse(nodeIdList: string[]): string {
+        // TODO This should also look at historical nodes
```

Comment:
Make into an issue?

Reply:
Done, after investigating this isn't an issue.